src/constants/sfx.asm

```diff
@@ -1,102 +1,109 @@
+;MUSIC_BOTWF                                    equ $45; Ballad of the Wind Fish, has triangle rhythm. More voices than $44
+;MUSIC_BOTWF                                    equ $46; Ballad of the Wind Fish, has added horn. More voices than $45
+;MUSIC_BOTWF                                    equ $47; Ballad of the Wind Fish, drums, high ting. Awakening tracks are not well described.
+MUSIC_GHOSTS_HOUSE                              equ $48
 MUSIC_ACTIVE_POWER_UP                           equ $49
-;MUSIC_SILENCE                                  equ $4A
-;MUSIC_GLITCHY                                  equ $4B
-;MUSIC_GLITCHY                                  equ $4C
-;MUSIC_SILENCE                                  equ $4D
-;MUSIC_SILENCE                                  equ $50
-;MUSIC_GLITCHY                                  equ $58
-;MUSIC_SILENCE                                  equ $59
-;MUSIC_GLITCHY                                  equ $5A
-;MUSIC_SILENCE                                  equ $5B
-;MUSIC_SILENCE                                  equ $5C
-;MUSIC_GLITCHY                                  equ $5D
-;MUSIC_SILENCE                                  equ $5E
-;MUSIC_SILENCE                                  equ $5F
-;MUSIC_SILENCE                                  equ $60
-MUSIC_COLOR_DUNGEON                             equ $61
-;MUSIC_UNTESTED                                 equ $6A
-;MUSIC_GLITCHED                                 equ $F0
-;MUSIC_NOTHING                                  equ $FF
+;MUSIC_SILENCE                                  equ $4A; More awakening...
+MUSIC_CATFISHS_MAW                              equ $4B
+;MUSIC_GLITCHY                                  equ $4C; Orchestral, short jingle.
+;MUSIC_SILENCE                                  equ $4D; harp jingle
+;MUSIC_MARIN_BEACH                              equ $4E
+;MUSIC_UNKNOWN                                  equ $4F; item jingle
+MUSIC_MINI_BOSS                                 equ $50
+;MUSIC_KANALET_CASTLE_COPY                      equ $51;	Copy of $13.
+;MUSIC_TAIL_CAVE_COPY                           equ $52;	Copy of $14.
+MUSIC_DREAM_SHRINE                              equ $53
+;MUSIC_UNKNOWN                                  equ $54; intense arpeggio, pre boss music? Switches to $19.
+MUSIC_FLYING_ROOSTER_REVIVED                    equ $55
+;MUSIC_UNKNOWN                                  equ $56; punch noises in upwards chord progression, plays $55 afterwards
+;MUSIC_CUCCO_HOUSE                              equ $57;	Is called 'Cucco House'
+MUSIC_FACE_SHRINE                               equ $58
+;MUSIC_WINDFISH                                 equ $59;	Is called 'The WindFish'
+MUSIC_TURTLE_ROCK                               equ $5A
+MUSIC_EAGLES_TOWER                              equ $5B
+MUSIC_BEFORE_EAGLES_TOWER_BOSS                  equ $5C;    Plays before the boss at the top of Eagle's Tower is summon by Grim Creeper.
```

Comment:
As a nice-to-have, if you still feel like polishing this:

For consistency, could you ensure that all comments follow the same format? Something like `MUSIC_CONSTANT     equ $VALUE ; Capitalized comment` (with a space before and after the semicolon)?

(But if you feel like you already worked much on this, I can merge it as-this.)

Reply:
I used a tab I was sure of my comment.
The lowercase ones are notes to help me quickly find tracks when I discover where they are used in the game. That's why some of them have/had oddly specific descriptions; if a particular sound evoked a really specific feeling I would be able to quickly tell where that track is in the list by looking for the note that describes the same feeling.
Once I'm done figuring all of them out, I will format it correctly.